pkg/gw/default_handlers.go

```diff
@@ -0,0 +1,2053 @@
+/*
+Copyright 2022 CodeNotary, Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gw
+
+import (
+	"context"
+	"io"
+	"net/http"
+
+	"github.com/codenotary/immudb/pkg/api/schema"
+	"github.com/codenotary/immugw/pkg/api"
+	immugwclient "github.com/codenotary/immugw/pkg/client"
+
+	"github.com/golang/protobuf/descriptor"
+	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/grpc-ecosystem/grpc-gateway/utilities"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/grpc/status"
+)
+
+// Suppress "imported and not used" errors
+var _ codes.Code
+var _ io.Reader
+var _ status.Status
+var _ = runtime.String
+var _ = utilities.NewDoubleArray
+var _ = descriptor.ForMessage
+var _ = metadata.Join
+
+// getClientForDb returns a client for the given database
+func getClientForDb(pathParams map[string]string, gwclient immugwclient.Client) (schema.ImmuServiceClient, error) {
+	databasename, ok := pathParams["databaseName"]
+	if !ok {
+		return nil, status.Errorf(codes.InvalidArgument, "missing parameter %s", "databaseName")
+	}
+	cli, err := gwclient.For(databasename)
+	if err != nil {
+		return nil, err
+	}
+	return cli.GetServiceClient(), nil
+}
+
+// request_ImmuService_ListUsers_0 is the handler for method ListUsers on the database
+func request_ImmuService_ListUsers_0(ctx context.Context, marshaler runtime.Marshaler, client schema.ImmuServiceClient, req *http.Request, pathParams map[string]string) (proto.Message, runtime.ServerMetadata, error) {
```

Comment:
@jeroiraz I could not find a better way to import default handlers from immudb because all of those methods expect the use of a single client, whereas in this scenario, we need multi db client. Please suggest if you think there is a better approach